company.el

```diff
@@ -1478,7 +1478,10 @@ end of the match."
     (snippet . "υ")
     (string . "φ")
     (struct . "Χ")
-    (variable . "ѱ")))
+    (variable . "ѱ"))
+  "List of text icons."
+  :type 'list
+  :group 'company)
```

Comment:
We don't add the `:group` parameter when it duplicates the automatically detected value, please just remove it.

Reply:
Removed.